update documents about CI chapter (#222)
pingcap/tidb-dev-guide: src/get-started/write-and-run-unit-tests.md

### Running tests for a pull request

> > If you are not joined the organization, you should wait for a memmber to comment with `/ok-to-test` to your pull request.

Reviewer: ```suggestion
> If you haven't joined the organization, you should wait for a member to comment with `/ok-to-test` to your pull request.
```
